src/cases/utils/utils.ts

```diff
@@ -57,11 +57,11 @@ export class Utils {
     });
   }
 
-  public static hasMatchedPermissions(featureVariation: FeatureVariation, jurisdictionId: string, caseType: string, userRoles: string[]): boolean {
-    if (featureVariation.jurisdiction === jurisdictionId && featureVariation.caseType === caseType) {
-      if (userRoles) {
-        return featureVariation.roles ? userRoles.some((userRole) =>
-          featureVariation.roles.some((role) => role === userRole)) : false;
+  public static hasMatchedJurisdictionAndCaseType(featureVariation: FeatureVariation, jurisdictionId: string, caseType: string): boolean {
+    if (featureVariation.jurisdiction === jurisdictionId) {
```

Comment:
@johnbenjamin-hmcts - given the function returns either `true` or `false`, I think you could rewrite this using a ternary statement. Also, the optional check (i.e. `?`) on Line 62 is somewhat redundant because Line 61 assumes `featureVariation` is truthy